README.md

```diff
@@ -55,19 +55,26 @@ contains example models in the [pipelines/models/](pipelines/models/) directory.
 The diagram above illustrates the architecture of the proof of concept. The two main categories of clusters are the core
 and near edge clusters.
 
+> [!NOTE]
+> The rest of this documentation assumes usage of ACM[^2] for cluster and application management, and we highly recommend
+> this approach.
+> If you choose to use a different approach, you'll find some information on how to deploy the inference container
+> application using standalone ArgoCD instances on the edge clusters, in the dedicated
+> [Bring Your Own Cluster Management document](docs/byo-mgmt-gitops.md).
```

Comment:
I've added this here and in docs/preparing_the_infrastructure.md to draw attention to the possibility while trying not to pollute the docs with what I (at least currently) consider a second class alternative approach.